jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,545 @@
+    "                ]\n",
+    "            }\n",
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\"merge\": {}},\n",
+    "        {\"clip\": {}}\n",
+    "\n",
+    "\n",
+    "    ],\n",
+    "    \"delivery\": {\n",
+    "        \"google_cloud_storage\": {\n",
+    "            \"bucket\": \"devrel-notebooks\",\n",
+    "            \"credentials\": GCP_CREDENTIALS,\n",
+    "            \"path_prefix\": \"basemaps-to-cloud/\"\n",
+    "        }\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Print the response object\n",
+    "---\n",
+    "Retrieve the response object as text to print out if there are no errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ORDERS_API_URL = 'https://api.planet.com/compute/ops/orders/v2'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "param_req = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "    order_params), auth=auth, headers={'content-type': 'application/json'})\n",
+    "\n",
+    "print(param_req.text)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Place the order\n",
+    "---\n",
+    "The following code places the order."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def place_order(order_params, auth):\n",
+    "    '''Place an order and return the order URL\n",
+    "\n",
+    "    Parameters: order_params (dict): order parameters\n",
+    "                auth (requests.auth.HTTPBasicAuth): authentication\n",
+    "    Return (str): URL of the order\n",
+    "    '''\n",
+    "    response = requests.post(ORDERS_API_URL, data=json.dumps(\n",
+    "        order_params), auth=auth, headers=headers)\n",
+    "    order_id = response.json()['id']\n",
+    "    order_url = ORDERS_API_URL + '/' + order_id\n",
+    "    return order_url"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_url = place_order(order_params, session.auth)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Poll for data success\n",
+    "---\n",
+    "When you're ready to pull the data, you can poll for when it's ready. Here, the polling is limited to a number of loops routine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def poll_for_success(order_url, num_loops=30):\n",
+    "    '''Polls for order success, fails after num_loops\n",
+    "\n",
+    "    Parameters: order_url (str): url for order\n",
+    "                num_loops (int): number of times to poll before failing\n",
+    "    Return (str): state of the order\n",
```

Comment:
Similar to what I said for the other notebook, this doesn't actually return anything, just prints out the state of the order. So, please remove the return line and mention in the docstring that it prints out the state of the order.